public/index.html

```diff
@@ -19,11 +19,43 @@ <h1 class="header">Black Jack</h1>
         <div class="dealer"></div>
         <div class="players"></div>
       </div>
-      <div class="controls">
-        <button type="button" class="btn" id="start">Start</button>
-      </div>
     </div>
 
+    <div class="modal">
+      <h2 class="modal__header">
+        Add at least one player to start a new game!
+      </h2>
+      <form class="modal__form" id="modal__form">
+        <div class="btn-container">
```

Comment:
This could be better named
```suggestion
        <div class="modal__form-controls">
```